### DISCUSSION TITLE
Is it possible to lock plugin versions in the CP?

### DISCUSSION THREAD

**Discussion by phrichards** [craftcms/cms#13260] [Q&A]
We're looking for a way to block updates to an individual plugin while the developer fixes a bug. I know you can lock a composer version and manage the updates through the CLI, but is there any way to do this in the CP? Ideally we'd like to just hide the update button for that plugin entirely.

**Comment by brandonkelly** [comment]
That’s not currently possible, but you can prevent **all** control panel-based updates by disabling the [`allowUpdates`](https://craftcms.com/docs/4.x/config/general.html#allowupdates) config setting.

**Reply by phrichards** [reply]
Dang! Would be cool if you could add this in the future, we want to allow updates to all of the other plugins but just lock this one and I'm sure we're not the only ones that have run into this issue.
